crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -227,6 +227,46 @@ impl Format<PyFormatContext<'_>> for FormatDanglingComments<'_> {
     }
 }
 
+/// Formats the dangling comments within a parenthetical, for example:
+/// ```python
+/// [  # comment
+///     1,
+///     2,
+///     3,
+/// ]
+/// ```
+pub(crate) fn parenthetical_comments(comments: &[SourceComment]) -> FormatParentheticalComments {
+    FormatParentheticalComments { comments }
+}
+
+pub(crate) struct FormatParentheticalComments<'a> {
+    comments: &'a [SourceComment],
+}
+
+impl Format<PyFormatContext<'_>> for FormatParentheticalComments<'_> {
+    fn fmt(&self, f: &mut Formatter<PyFormatContext>) -> FormatResult<()> {
+        let mut first = true;
+
+        for comment in self
+            .comments
+            .iter()
+            .filter(|comment| comment.is_unformatted())
+        {
+            if first {
+                write!(f, [space(), space()])?;
+            }
```

Comment:
Is it possible for a `(` to have more than one dangling comment? 

Either way, this results in `(  # first#second` if a `(` happens to have more than one comment which isn't ideal. I think we should just remove the `first` check, similar to how it is done in `trailing_comments`

Reply:
that would be the main question for me to be able to review this PR

Reply:
It's not possible, no.